hi-ml-multimodal/notebooks/phrase_grounding.ipynb

```diff
@@ -156,12 +156,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Inference\n",
     "\n",
-    "We will run inference on a chest X-ray from [Radiopaedia](https://radiopaedia.org/), but any other chest X-ray image in DICOM or JPEG format can be used for research purposes."
+    "We will run inference on a chest X-ray from the [Open-i dataset](https://openi.nlm.nih.gov/detailedresult?img=CXR111_IM-0076-1001&req=4), but any other chest X-ray image in DICOM or JPEG format can be used for research purposes."
```

Comment:
```suggestion
    "We will run inference on a chest X-ray from [Open-i](https://openi.nlm.nih.gov/detailedresult?img=CXR111_IM-0076-1001&req=4), but any other chest X-ray image in DICOM or JPEG format can be used for research purposes."
```
Trying to avoid getting into the surprisingly complex semantics of what "Open-i" actually is (database v. dataset, also whether we want to specifically refer to the Indiana University CXR subset within it, etc.).